lib/recurly/fraud.js

```diff
@@ -130,7 +138,7 @@ export class Fraud extends Emitter {
       return this._form = form;
     } else {
       const missingFormError = errors('fraud-data-collector-missing-form', { selectors });
-      return this.emit('error', missingFormError);
+      this.emit('error', missingFormError);
```

Comment:
I removed this return so that an error would return undefined instead. This was revealed in the course of adding tests.